### ISSUE TITLE
Binder Error: Cannot extract field from expression "aminoAcidInsertions.*" because it is not a struct

### ISSUE THREAD

**Issue by corneliusroemer** [GenSpectrum/LAPIS-SILO#457]
```
data.ndjson.zst is not empty (2299313 bytes), starting preprocessing
[2024-05-30 16:59:54.470] [logger] [info] [api.cpp:312] Starting SILO
[2024-05-30 16:59:54.471] [logger] [info] [api.cpp:272] Starting SILO preprocessing
[2024-05-30 16:59:54.471] [logger] [info] [yaml_file.cpp:11] Reading config from ./default_preprocessing_config.yaml
[2024-05-30 16:59:54.471] [logger] [info] [yaml_file.cpp:11] Reading config from ./preprocessing_config.yaml
[2024-05-30 16:59:54.471] [logger] [info] [api.cpp:70] Resulting preprocessing config: { input directory: '/preprocessing/input/', pango_lineage_definition_file: 'pangolineage_alias.json', output_directory: '/preprocessing/output/', metadata_file: 'none', reference_genome_file: 'reference_genomes.json',  gene_file_prefix: 'gene_',  nucleotide_sequence_file_prefix: 'nuc_', unalgined_nucleotide_sequence_file_prefix: 'unaligned_', ndjson_filename: 'data.ndjson.zst', preprocessing_database_location: none }
[2024-05-30 16:59:54.471] [logger] [info] [database_config.cpp:212] Reading database config from database_config.yaml
[2024-05-30 16:59:54.473] [logger] [info] [api.cpp:255] preprocessing - reading reference genome
[2024-05-30 16:59:54.473] [logger] [info] [reference_genomes.cpp:148] Read reference genomes from file: /preprocessing/input/reference_genomes.json
[2024-05-30 16:59:54.473] [logger] [info] [api.cpp:259] preprocessing - reading pango lineage alias
[2024-05-30 16:59:54.473] [logger] [info] [pango_lineage_alias.cpp:126] Read pango lineage alias from file: /preprocessing/input/pangolineage_alias.json
[2024-05-30 16:59:54.478] [logger] [info] [preprocessor.cpp:44] preprocessing - creating intermediate results directory '/preprocessing/temp/'
[2024-05-30 16:59:54.478] [logger] [info] [preprocessor.cpp:60] preprocessing - ndjson pipeline chosen
[2024-05-30 16:59:58.951] [logger] [info] [preprocessor.cpp:250] preprocessing - skip partition merging because no partition_by key was provided, instead putting all sequences into the same partition
[2024-05-30 17:00:04.917] [logger] [error] [api.cpp:280] Binder Error: Cannot extract field from expression "aminoAcidInsertions.*" because it is not a struct
[2024-05-30 17:00:04.917] [logger] [info] [api.cpp:317] Stopping SILO
std::exception
Command exited with non-zero status 70
```

**Comment by corneliusroemer** [comment]
https://<redacted>.slack.com/files/U05FBBSP02F/F075V40B3NF/input.tar.zst
https://<redacted>.slack.com/files/U05FBBSP02F/F075ELA3Y5D/config.tar.zst

**Comment by Taepper** [comment]
Resolved by #464
